src/components/Popups/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -348,4 +317,41 @@ describe('components/Popups/ReplyField', () => {
             expect(setActiveItemSpy).toBeCalledWith(0);
         });
     });
+
+    describe('restoreEditor()', () => {
+        test('should restore value and cursor position', () => {
+            const wrapper = getWrapper({ cursorPosition: 1, value: 'test' });
+
+            createMentionSelectorState.mockImplementationOnce((value: string) =>
+                EditorState.createWithContent(ContentState.createFromText(value)),
+            );
+            const editorState = wrapper.instance().restoreEditor();
+
+            expect(editorState.getCurrentContent().getPlainText()).toEqual('test');
+            expect(editorState.getSelection().getFocusOffset()).toEqual(1);
+        });
+
+        test('should reset cursor position if value is empty', () => {
+            const wrapper = getWrapper({ cursorPosition: 1 });
+
+            const editorState = wrapper.instance().restoreEditor();
+
+            expect(editorState.getSelection().getFocusOffset()).toEqual(0);
+        });
+    });
+
+    describe('render()', () => {
+        test('should render the editor with right props', () => {
+            const wrapper = getWrapper();
+
+            expect(wrapper.prop('className')).toBe('ba-Popup-field ba-ReplyField');
+        });
+
+        test('should render PopupList if popupreference is not null', () => {
```

Comment:
I know this was existing, but would it make sense to add a test to check that `PopupList` doesn't exist if `popupReference` is null?